samples/Ch18_using_libs/fig_18_14_pstl_usm_device.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2023 Intel Corporation
+
+// SPDX-License-Identifier: MIT
+
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/algorithm>
+#include <sycl/sycl.hpp>
+
+int main() {
+  sycl::queue Q;
+  const int n = 10;
+  int * h_head = sycl::malloc_host<int>(n, Q);
+  int * d_head = sycl::malloc_device<int>(n, Q);
+  std::fill(oneapi::dpl::execution::make_device_policy(Q),
+            d_head, d_head + n, 78);
+  Q.wait();
+
+  Q.memcpy(h_head, d_head, n*sizeof(int));
```

Comment:
This is debatably personal preference, feel free to ignore, but the templated `copy` function is a little more concise.

```suggestion
  Q.copy(d_head, h_head, n);
```